benchexec/containerexecutor.py

```diff
@@ -976,7 +976,10 @@ def check_child_exit_code():
 
             # start measurements
             cgroups.add_task(grandchild_pid)
-            parent_setup = parent_setup_fn()
+            parent_setup = parent_setup_fn(
+                child_pid=child_pid,
+                grandchild_pid=grandchild_pid,
```

Comment:
Do we need both? `child_pid` and the fact that there are two processes are really implementation details, and we shouldn't expose them in a public API. Would it be enough if we simply pass the PID of the benchmarked process?